samples/highcharts/chartchooser/categorical-hierarchy-sunburst-monochrome/demo.js

```diff
@@ -0,0 +1,239 @@
+const data = [
+    {
+        id: "Happy",
+        parent: "",
+        name: "Happy",
+        color: "#FFFFFF"
+    },
+    {
+        id: "Playful",
+        parent: "Happy",
+        name: "Playful"
+    },
+    {
+        id: "Content",
+        parent: "Happy",
+        name: "Content"
+    },
+    {
+        id: "Interested",
+        parent: "Happy",
+        name: "Interested"
+    },
+    {
+        id: "Proud",
+        parent: "Happy",
+        name: "Proud"
+    },
+    {
+        id: "Accepted",
+        parent: "Happy",
+        name: "Accepted"
+    },
+    {
+        id: "Powerful",
+        parent: "Happy",
+        name: "Powerful"
+    },
+    {
+        id: "Peaceful",
+        parent: "Happy",
+        name: "Peaceful"
+    },
+    {
+        id: "Trusting",
+        parent: "Happy",
+        name: "Trusting"
+    },
+    {
+        id: "Optimistic",
+        parent: "Happy",
+        name: "Optimistic"
+    },
+
+    /* Level 3 */
+    {
+        id: "Aroused",
+        parent: "Playful",
+        name: "Aroused",
+        value: 1
+    },
+    {
+        id: "Checky",
+        parent: "Playful",
+        name: "Checky",
+        value: 1
+    },
+
+    {
+        id: "Free",
+        parent: "Content",
+        name: "Free",
+        value: 1
+    },
+    {
+        id: "Joyful",
+        parent: "Content",
+        name: "Joyful",
+        value: 1
+    },
+
+    {
+        parent: "Interested",
+        name: "Curious",
+        value: 1
+    },
+    {
+        parent: "Interested",
+        name: "Inquisitive",
+        value: 1
+    },
+
+    {
+        parent: "Proud",
+        name: "Successful",
+        value: 1
+    },
+    {
+        parent: "Proud",
+        name: "Confident",
+        value: 1
+    },
+
+    {
+        parent: "Accepted",
+        name: "Respected",
+        value: 1
+    },
+    {
+        parent: "Accepted",
+        name: "Valued",
+        value: 1
+    },
+
+    {
+        parent: "Powerful",
+        name: "Courageous",
+        value: 1
+    },
+    {
+        parent: "Powerful",
+        name: "Creative",
+        value: 1
+    },
+
+    {
+        parent: "Peaceful",
+        name: "Loving",
+        value: 1
+    },
+    {
+        parent: "Peaceful",
+        name: "Thankful",
+        value: 1
+    },
+
+    {
+        parent: "Trusting",
+        name: "Sensitive",
+        value: 1
+    },
+    {
+        parent: "Trusting",
+        name: "Intimate",
+        value: 1
+    },
+
+    {
+        parent: "Optimistic",
+        name: "Hopeful",
+        value: 1
+    },
+    {
+        parent: "Optimistic",
+        name: "Inspired",
+        value: 1
+    }
+];
+
+// Splice in transparent for the center circle
+Highcharts.getOptions().colors.splice(0, 0, "transparent");
+Highcharts.setOptions({
+    colors: ["#B0B0B0", "#FFFFFF"]
+});
```

Comment:
Can be combined. Just set `transparent` as the first in the array.

Reply:
Done 👍